Resources/views/frontend/_public/src/js/applepay.js

```diff
@@ -16,8 +16,17 @@
      */
     function hideApplePayIfNotAllowed() {
         // Find the hidden Apple Pay element
-        var applePayInput = document.getElementsByClassName('payment-mean-mollie-applepay');
-        var applePayLabel = document.getElementsByClassName('payment-mean-mollie-applepay-label');
+        var applePayInput = document.querySelector('input.payment-mean-mollie-applepay');
+        var applePayLabel = document.querySelector('label.payment-mean-mollie-applepay-label');
+
+        if (typeof applePayInput === 'undefined' || !applePayLabel) {
```

Comment:
could you add comments what this actually does as a fallback detection?
i doubt anyone will remember this here otherwise